roles/ldap_kerberos/templates/kdc.conf.j2

```diff
@@ -0,0 +1,55 @@
+includedir /etc/krb5.conf.d/
+
+[logging]
+ default = FILE:/var/log/krb5libs.log
+ kdc = FILE:/var/log/krb5kdc.log
+ admin_server = FILE:/var/log/kadmind.log
+
+[libdefaults]
+ dns_lookup_realm = false
+ ticket_lifetime = 24h
+ renew_lifetime = 7d
+ forwardable = true
+ rdns = false
+ pkinit_anchors = FILE:/etc/pki/tls/certs/ca-bundle.crt
+ default_realm = {{ realm }}
+ default_ccache_name = /tmp/krb5cc_%{uid}
+ canonicalize = true
```

Comment:
Some of the variables are duplicated between `kdc.conf` and `krb5.conf`.
The kdc will merge `krb5.conf` and `kdc.conf`. Therefore there should be only the different / needed variables in `kdc.conf`.

https://web.mit.edu/kerberos/krb5-1.12/doc/admin/conf_files/kdc_conf.html

Reply:
Ok so according to the doc, we should put common configuration between client and server in `krb5.conf` and put in `kdc.conf` only stuff related to KDC server right ?

Reply:
That's right

Reply:
done